relay-server/src/services/processor/profile.rs

```diff
@@ -72,8 +72,18 @@ pub fn transfer_id(state: &mut ProcessEnvelopeState<TransactionGroup>) {
 #[cfg(feature = "processing")]
 pub fn process(state: &mut ProcessEnvelopeState<TransactionGroup>, config: &Config) {
     let profiling_enabled = state.project_state.has_feature(Feature::Profiling);
+    let continuous_profiling_enabled = state
+        .project_state
+        .has_feature(Feature::ContinuousProfiling);
```

Comment:
Let's also add this check to `filter` so we can drop continuous profiles in PoPs.